2wp-app simplify the documentation (#929)
rsksmart/devportal: content/rsk-devportal/guides/two-way-peg-app/faqs.md

---
menu_order: 1300
menu_title: FAQs
title: "Frequently Asked Questions (FAQs) | 2 way peg app Documentation"
description: "Frequently asked questions."
tags: 2 way peg, powpeg, peg-in, peg-out, 2way-peg, bridge, rbtc, btc, rootstock, testnet, mainnet, guide, setup, integrate, use
layout: rsk
---

Here, you can find a list of frequently asked questions (FAQs) about the 2 way peg application.

**What are the requirements to use 2wp-app?**
- To know more about the requirements, see [requirements](/guides/two-way-peg-app/requirements/).

**What are the commonm errors in pegout transactions?**
- To know more about the common errors, see [common errors](/guides/two-way-peg-app/pegout/pegout-common-errors).

**What is the difference between SegWit and Legacy addresses?**
- Legacy address is the original BTC address while SegWit is the newer address format with lower fees. SegWit means Segregated Witness, where Segregated is to separate and Witness is the transaction signatures involved with a specific transaction.

**What type of addresses do I need to perform a peg in?**

> - For information on the type of addresses to use when performing a peg-in transaction, see [supported addresses](/guides/two-way-peg-app/supported-addresses) page.

Reviewer: ```suggestion
- For information on the type of addresses to use when performing a peg-in transaction, see [supported addresses](/guides/two-way-peg-app/advanced-operations/supported-addresses) page.
```

Here, you can find a list of frequently asked questions (FAQs) about the 2 way peg application.

**What are the requirements to use 2wp-app?**
- To know more about the requirements, see [requirements](/guides/two-way-peg-app/requirements/).

> **What are the commonm errors in pegout transactions?**

Reviewer 2: ```suggestion
**What are the common errors in peg-out transactions?**
```

## Resources
- 2 way peg app frontend [repo](https://github.com/rsksmart/2wp-app)
- 2 way peg app backend [repo](https://github.com/rsksmart/2wp-api)

> - How to get [RBTC using RSK’s built in Powpeg](https://developers.rootstock.io/guides/get-crypto-on-rsk/powpeg-btc-rbtc/)

Reviewer 2: ```suggestion
- How to get [RBTC using Rootstock's built in Powpeg](https://developers.rootstock.io/guides/get-crypto-on-rsk/powpeg-btc-rbtc/)
```
